drivers/tcu_pynq/architecture.py

```diff
@@ -55,8 +55,8 @@
     array_size=32,
     dram0_depth=2097152,
     dram1_depth=2097152,
-    local_depth=16384,
-    accumulator_depth=4096,
+    local_depth=49152,
```

Comment:
this is due to enabling uram?

Reply:
Yes. Also to reflect the change to zcu104.tarch file in the previous PR.